### PR TITLE
fix words issue

### PR DESCRIPTION
**Pull request by helight**
[KubeletConfiguration file] should be  [Kubelet Configuration file]

### CODE REVIEW

**Comment by helight** [comment]
> /check-cla
> @helight if you have any problem for CLA we can help you

thank you, I am trying to fix it, can you help to find what is wrong with my cncf account?

image

**Comment by helight** [comment]
@k8s-ci-robot I signed it

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can configure a static Pod with either a [file system hosted configuration f
 
 ### Filesystem-hosted static Pod manifest {#configuration-files}
 
-Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [KubeletConfiguration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
+Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [Kubelet Configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: nit:
```suggestion
Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ For example, this is how to start a simple web server as a static Pod:
     ```
     KUBELET_ARGS="--cluster-dns=10.254.0.10 --cluster-domain=kube.local --pod-manifest-path=/etc/kubelet.d/"
     ```
-    or add the `staticPodPath: <the directory>` field in the [KubeletConfiguration file](/docs/tasks/administer-cluster/kubelet-config-file).
+    or add the `staticPodPath: <the directory>` field in the [Kubelet Configuration file](/docs/tasks/administer-cluster/kubelet-config-file).
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: nit:
```suggestion
    or add the `staticPodPath: <the directory>` field in the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file).
```

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: ok

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 2 above)
Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: ok

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ You can configure a static Pod with either a [file system hosted configuration f
 
 ### Filesystem-hosted static Pod manifest {#configuration-files}
 
-Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [Kubelet Configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
+Manifests are standard Pod definitions in JSON or YAML format in a specific directory. Use the `staticPodPath: <the directory>` field in the [kubelet configuration file](/docs/tasks/administer-cluster/kubelet-config-file), which periodically scans the directory and creates/deletes static Pods as YAML/JSON files appear/disappear there.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: @helight , thanks for the updates.
What do you think about these possible edits for lines 55 - 56:

Manifests are standard Pod definitions in JSON or YAML format.
You can define a static Pod by creating a [kubectl configuration file](/docs/tasks/administer-cluster/kubelet-config-file) and setting the `staticPodPath` field to a directory containing your static Pod manifests.
The kubelet periodically scans this directory and creates or deletes the static Pods as the directory contents change.

Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: so, what for? I don't thinks it is fix any problem. it has the same meaning with origin words.

Comment (kbhawkey) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: hello @helight. As a native English speaker/reader, I found the second sentence somewhat unclear.
I tried to answer the question: What (which) periodically scans the directory and creates ...

Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: yes, you are right about the sentence. it should make clear that kubelet scan the file.
but in fact I can understand what this origin sentence talk about, I know the [kubelet configuration file] should be scaned by kubelet.
so I not sure it should be modified or not.

Comment (sftim) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: @kbhawkey I think your proposed wording will be easier for localization teams to work with too.

Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: @sftim so are you agree we should modified it as @kbhawkey 's fix

Comment (kbhawkey) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: @helight, your changes seem fine. If time, I will create another PR to modify the sentence structure. Thanks!

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-18312--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/static-pod/

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 3 above)
Comment (helight) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: @kbhawkey ok, thanks.
 If there is anything I can do, just tell me, I am glad to do that.

**Pull request merged**